apollo-federation/tests/subgraph/subgraph_validation_tests.rs

```diff
@@ -307,14 +309,13 @@ mod fieldset_based_directives {
                 ),
                 (
                     "EXTERNAL_ON_INTERFACE",
-                    r#"[S] Interface type field "T.f" is marked @external but @external is not allowed on interface fields (it is nonsensical)."#,
+                    r#"[S] Interface type field "T.f" is marked @external but @external is not allowed on interface fields."#,
```

Comment:
This error did match, but the test is failing since `UNSUPPORTED_ON_INTERFACE` error is still missing.